pom.xml

```diff
@@ -261,9 +263,13 @@
           <plugin>
             <groupId>org.jacoco</groupId>
             <artifactId>jacoco-maven-plugin</artifactId>
-            <version>0.8.1</version>
             <configuration>
               <output>file</output>
+              <excludes>
+                <exclude>**/*.sketch</exclude>
+                <exclude>**/org/cactoos/**/*</exclude>
+                <exclude>**/ghman/data/*</exclude>
```

Comment:
@olenagerasimova why is this excluded?

Reply:
@victornoel it's database related package, I do not think there will be any unit tests for classes of this package. But actually it might be unfair to exclude it from coverage metrics. @g4s8 what do say? Should we exclude it or not?

Reply:
@olenagerasimova I don't see why there would be a different treatment for this particular package. Let's wait for @g4s8 ruling on this anyway :)

Reply:
@victornoel @olenagerasimova I think we can test `data` package with integration tests and then add it back to coverage metrics, WDYT?

Reply:
@g4s8 @olenagerasimova I just don't understand why we would need to first exclude them.

Reply:
@victornoel @g4s8 I just remove this line to save our time